host/bcm283x/gpio.go

```diff
@@ -368,18 +368,28 @@ func (p *Pin) PWM(duty gpio.Duty, period time.Duration) error {
 		return nil
 	}
 
-	// TODO(maruel): Leverage oversampling.
-	if _, _, err := clockMemory.pwm.set(uint64(time.Second/period), 1); err != nil {
+	base_freq := uint64(250 * 1000 * 1000) // 250MHz
```

Comment:
Please keep the TODO, I have a local branch to eventually support approximative clocks via oversampling.

Reply:
Reverted.